homeassistant/components/light/avion.py

```diff
@@ -10,14 +10,14 @@
 import voluptuous as vol
 
 from homeassistant.const import CONF_API_KEY, CONF_DEVICES, CONF_NAME, \
-    CONF_USERNAME, CONF_PASSWORD
+    CONF_USERNAME, CONF_PASSWORD, CONF_ID
 
 from homeassistant.components.light import (
     ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light,
     PLATFORM_SCHEMA)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['avion==0.7']
+REQUIREMENTS = ['antsar-avion==0.9']
```

Comment:
Why are you changing the requirement?

Reply:
@fabaff thanks for noticing/asking. I'm curious if I've gone about this the right way.

I've opened a PR making the necessary changes to `avion`. Since this vendor API change breaks the library, I thought I'd switch to my fork in the meantime to keep Home Assistant working. The author seems like a busy guy (my last PR took a few months to get merged).

`avion` PR: https://github.com/mjg59/python-avion/pull/9
Previous `avion` PR : https://github.com/mjg59/python-avion/pull/8 (merged, but I later realized the fix was incomplete)

Is switching a bad way to handle this? If it's preferable to wait for the `avion` PR to get merged, I'm ok with that.

Reply:
> Is switching a bad way to handle this?

No, can be done especially If bugs are not addressed or the developer/maintainer is non-responsive. If @mjg59 is busy then we can go on with your fork. Would be nice if you could address those two things in your fork:

- https://github.com/mjg59/python-avion/issues/10
- https://github.com/mjg59/python-avion/issues/11

Also, I would like to suggest that you update the README in your fork with to make it clear that's a fork.

Reply:
@antsar Instead of changing to your fork you could ask to be added as a collaborator to the main repo. Of course this depends on how @mjg59 feels about this.